PropTechAPI - コピー/src/main/java/com/PropTech/UserRegist/PropTechAppAPI/PropTechAPI/Logic/AWSCognito.java

```diff
@@ -0,0 +1,15 @@
+package com.example.AWSCodeGuru.TestCode;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.ayth.BasicAWSCredentials;
+
+static String myKeyId = "AKIAX742FUDUQXXXXXXX";
+static String mySecretKey = "MySecretKey";
+
+public static void main(String[] args){

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, AnnotationTypeDeclarationHeader expected instead